about.html

```diff
@@ -35,7 +35,10 @@ <h1 class="display-4"><strong>2.2 by 22'</strong></h1>
 
 <div class="container">
   <div class="row">
-    <div class="col-sm-4">
+    <div class="col-md order-md-2 order-1">
```

Comment:
you were getting that weird result because every row is automatically split into 12 sections by default, so when you chose `col-sm-4` you were defining a column that took up 4/12 (1/3) of the row on everything larger than the smallest possible screens.

By choosing `col-md` instead, with no number, I'm telling it to make equal-length columns with any other columns in this row. I could have done `col-md-6` for the same result here.

by putting 'md' in there, I am telling bootstrap that I only want it to make columns on medium screens OR LARGER. On smaller screens (like phones), it will stack the columns on top of each other. 

The two order classes fix the order of the stacking. order-1 says "put this first by default" while order-md-2 say "put this second on medium screens or larger". This has the effect of making the column be on the RIGHT when you have larger screens, but on TOP if you have smaller screens.

### Larger Screens
![image](https://user-images.githubusercontent.com/49095734/93952342-f6390680-fd1e-11ea-959c-cfc038db1dac.png)

### Smaller Screens
<img src="https://user-images.githubusercontent.com/49095734/93952367-094bd680-fd1f-11ea-9962-7fa2a88d2512.png" width=50%>